### ISSUE TITLE
Cross entropy 輸入的input有誤？

### ISSUE THREAD

**Issue by xup6YJ** [Jiawei0o0/mutual-learning-with-reliable-pseudo-labels#2]
作者您好
您算cross entropy loss都是用softmax過後的機率進行計算
不過nn.crossentropy本身就有softmax了

**Comment by youngsnpeak** [comment]
你看错了，写的没问题，代码里用model output和one_hot标签之间计算的；有标记数据CE损失用的F.ce(虽然没用到)，算得也没错

**Comment by xup6YJ** [comment]
感謝您的回覆！
